featuretools/tests/primitive_tests/test_upper_case_word_count.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+import pandas as pd
+
+from featuretools.primitives import UpperCaseWordCount
+
+
+class TestUpperCaseWordCount:
+    primitive = UpperCaseWordCount
+
+    def test_strings(self):
+        x = pd.Series(
+            [
+                "This IS a STRING.",
+                "Testing AAA",
+                "Testing AAA-BBB",
```

Comment:
I think in the change @sbadithe just made to `NumWords` the text `AAA-BBB` would be treated as a single word, not two words like we are doing in this primitive. We should probably reconcile that difference.

Reply:
I _think_ this can be done by replacing `\b` with `\w` in the regex

Reply:
Fixed:
https://github.com/alteryx/featuretools/pull/2460/commits/7b162d66842596a2eb47407388ad6c896c7253ce